typescript/sdk/src/utils/cosmos.ts

```diff
@@ -431,18 +741,22 @@ export const CosmosChainSchema = z
       .array(
         z
           .object({
-            kind: z.string().optional(),
-            url: z.string().optional(),
-            tx_page: z.string().optional(),
-            account_page: z.string().optional(),
+            kind: z.string().min(1).optional(),
+            url: z.string().min(1).optional(),
+            tx_page: z.string().min(1).optional(),
+            account_page: z.string().min(1).optional(),
+            validator_page: z.string().min(1).optional(),
+            proposal_page: z.string().min(1).optional(),
+            block_page: z.string().min(1).optional(),
           })
           .strict(),
       )
       .optional(),
-    keywords: z.array(z.string()).optional(),
+    keywords: z.array(z.string().min(1)).optional(),
     extra_codecs: z.array(z.enum(['ethermint', 'injective'])).optional(),
   })
-  .passthrough()
```

Comment:
can you keep `passthrough()`, we intentionally added this to be robust to schema additions on their side

Reply:
a comment about this would help too

Reply:
spoke offline but 

passthrough vs strict.and(intersection(any,any)) should be the same thing, will leave a comment